src/Init/Core.lean

```diff
@@ -1612,6 +1612,11 @@ class Antisymm {α : Sort u} (r : α → α → Prop) where
 namespace Lean
 /-! # Kernel reduction hints -/
 
+/--
+A placeholder axiom so `#print axioms` can report use of `reduceBool` and `reduceNat`.
+-/
+axiom trustedCodeBase : False
```

Comment:
This isn't a great name for an axiom, it's like naming an axiom `axiom`. Maybe `trustTheCompiler` is a better name, or `trustCompilerAndImplementedBy`...